waku/v2/protocol/filter/filter_push_test.go

```diff
@@ -141,3 +144,130 @@ func (s *FilterTestSuite) TestLargePayloadsUTF8() {
 	s.Require().NoError(err)
 
 }
+
+func (s *FilterTestSuite) TestEmptyPayload() {
+
+	// Subscribe
+	s.subDetails = s.subscribe(s.testTopic, s.testContentTopic, s.fullNodeHost.ID())
+
+	// Should get rejected
+	_, err := s.relayNode.Publish(s.ctx, tests.CreateWakuMessage(s.testContentTopic, utils.GetUnixEpoch(), ""), relay.WithPubSubTopic(s.testTopic))
```

Comment:
This is not a test for filter, because this is testing publish for a relayNode. Ideally we should have this test under relay tests.

Reply:
There is no way how lightNode(filter) could send PUSH message, correct ?

Reply:
No, lightNode is a receiver of PUSH messaage. That can only be sent by a filterFullNode.

If you want to simulate pushMessage, you will have to use an internal method from filterFullNode `pushMessage`. 

But as my question remains, what are we trying to validate here? Because filterPush is not an API someone can use to push a message. It is automatically done by a filterFullNode on receiving any message on relay that is of interest to a filter subscriber.

Reply:
Understand, I would suggest to move negative tests to relay module and confirm positive tests with reception on the filter side. Does it look reasonable ?